styles/fonts.css

```diff
@@ -0,0 +1,12 @@
+@font-face {
+font-family: 'Gotham Pro';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/r4/pullRequest?prid=5887703)